### ISSUE TITLE
Bugreport: propery merge fails when using multiple properties with multiple values

### ISSUE THREAD

**Issue by koenpunt** [jakubpawlowicz/clean-css#655]
My css
```css
#layer1 {
  background: url('../images/house.png') no-repeat center 1px;
  background: url('../images/house_01.png') no-repeat center 1px,
              url('../images/house_02.png') no-repeat center 1448px,
              url('../images/house_03.png') no-repeat center 2596px,
              url('../images/house_04.png') no-repeat center 3743px;
}
```

Which results in:

```css
#layer-1{background:url(../images/house_01.png),url(../images/house_02.png),url(../images/house_03.png),url(../images/house_04.png) center 1px,center 1448px,center 2596px,center 3743px/auto,auto,auto,auto no-repeat,no-repeat,no-repeat,no-repeat;}
```

Reformatted:

```css
#layer-1 {
   background: url(../images/house_01.png), 
               url(../images/house_02.png),
               url(../images/house_03.png),
               url(../images/house_04.png) center 1px,
               center 1448px,
               center 2596px,
               center 3743px/auto,
               auto,
               auto,
               auto no-repeat,
               no-repeat,
               no-repeat,
               no-repeat;
}
```

When I remove the first `background` property, the minified result is correct, so somewhere in the merging of tags this problem arises.

For now I disabled the `advanced` setting.

**Comment by jakubpawlowicz** [comment]
There's certainly something wrong about it. I will take a closer look soon.

**Comment by jakubpawlowicz** [comment]
@koenpunt - it's fixed in 3.4.2 now. Thanks for reporting it.

**Comment by koenpunt** [comment]
Thanks! :+1:
